node/src/blockchain/blockchains.rs

```diff
@@ -0,0 +1,414 @@
+// Copyright (c) 2019-2021, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::blockchain::blockchain_interface::{
```

Comment:
Is there a specific need to introduce another file beyond `blockchain_interface.rs` and `blockchain_bridge.rs` to resolve a circular dependency or another similar reason?